sdk/tracing/azure-core-tracing-opencensus/src/samples/java/com/azure/core/tracing/opencensus/PublishEvents.java

```diff
@@ -0,0 +1,85 @@
+package com.azure.core.tracing.opencensus;
+
+import com.azure.core.util.Context;
+import com.azure.messaging.eventhubs.*;
```

Comment:
I thought there was a checkstyle rule about `*` imports

Reply:
This check might be skipped on the samples files, I'll remove the `*`